Jenkinsfile

```diff
@@ -231,7 +231,7 @@ node {
     }
   }
 
-  builders['macOS'] = get_macos_pipeline()
+  //builders['macOS'] = get_macos_pipeline()
```

Comment:
I think it is best to keep this enabled. The mac build usually passes on at least one of the two builds which Jenkins does, and we can go ahead and merge in this case.